client/src/components/lower-info-area/lower-container.vue

```diff
@@ -128,9 +128,8 @@ import { Component, Vue } from 'vue-property-decorator'
 })
 export default class LowerContainer extends Vue {
   private needMoreInfo = 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/' +
-                          'permits-licences/businesses-incorporated-companies/approval-business-name'
+                         'permits-licences/businesses-incorporated-companies/approval-business-name'
```

Comment:
It's too bad with urls you can't template string them, and do without the concatenation... or can you?!

Reply:
Just tried. You can't (it adds spaces).